connectors/automation-anywhere/element-templates/automation-anywhere-connector.json

```diff
@@ -85,8 +85,8 @@
       "constraints": {
         "notEmpty": true,
         "pattern": {
-          "value": "^(=|http://|https://|secrets).*$",
-          "message": "Must be a http(s) URL."
+          "value": "^(=|http://|https://|\\{\\{secrets).*$",
+          "message": "Must be a http(s) URL or contain a secret reference <a href=\"https://docs.camunda.io/docs/components/connectors/use-connectors/#using-secrets/” target=\"_blank\">a secret reference</a>\"."
```

Comment:
What about adding the closing parentheses? 
```
^(=|http:\/\/|https:\/\/).*$|{{secrets.*}}
```

Reply:
If the user forgets the closing parentheses it will lead to an error so why not requiring users to use the full pattern here.